brazilfiscalreport/danfe/danfe.py

```diff
@@ -773,9 +773,17 @@ def _draw_recipient_sender(self):
             dest_cnpj_cpf = extract_text(self.dest, "CPF")
         dest_cnpj_cpf = format_cpf_cnpj(dest_cnpj_cpf)
         date_emi, time_emi = get_date_utc(extract_text(self.ide, "dhEmi"))
-        dest_end = (
-            f"{extract_text(self.dest,'xLgr')}, " f"{extract_text(self.dest,'nro')}"
-        )
+        complemento = extract_text(self.dest, "xCpl")
+        if complemento:
+            dest_end = (
+                f"{extract_text(self.dest,'xLgr')}, "
+                f"{extract_text(self.dest,'nro')}, "
+                f"{extract_text(self.dest,'xCpl')}"
+            )
+        else:
+            dest_end = (
+                f"{extract_text(self.dest,'xLgr')}, " f"{extract_text(self.dest,'nro')}"
+            )

```

Comment:
```suggestion
logradouro = extract_text(self.dest, "xLgr")
numero = extract_text(self.dest, "nro")
complemento = extract_text(self.dest, "xCpl")

partes = [logradouro, numero]
if complemento:
    partes.append(complemento)

dest_end = ", ".join(partes)
```

poderia ser feito assim